charts/shoot-core/components/charts/apiserver-proxy/templates/apiserver-proxy-pod-mutatingwebhook.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: {{ include "webhookadmissionregistration" . }}
+kind: MutatingWebhookConfiguration
+metadata:
+  name: apiserver-proxy.networking.gardener.cloud
+  annotations:
+    networking.gardener.cloud/description: |-
```

Comment:
```suggestion
    gardener.cloud/description: |-
```

Reply:
We already have the following:

```yaml
resources.gardener.cloud/description: |-
  DO NOT EDIT - This resource is managed by gardener-resource-manager.
  Any modifications are discarded and the resource is returned to the original state.
```

it's better not to use the `gardener.cloud/description` as it might be used for other things in the future.

Reply:
Agreed